cli/src/pcluster/templates/cdk_builder.py

```diff
@@ -45,13 +45,12 @@ def build_cluster_template(
             ClusterCdkStack(app, output_file, stack_name, cluster_config, bucket, log_group_name)
 
             cloud_assembly = app.synth()
+            LOGGER.info("CDK template generation completed successfully")
 
             cdk_artifacts_manager = CDKArtifactsManager(cloud_assembly)
             assets_metadata = cdk_artifacts_manager.upload_assets(bucket=bucket)
             generated_template = cdk_artifacts_manager.get_template_body()
```

Comment:
Nitpick: do we want to add a log about the upload process?

Reply:
Yes, it's already logged in the [CDKArtifactsManager](https://github.com/EddyMM/aws-parallelcluster/blob/develop/cli/src/pcluster/templates/cdk_artifacts_manager.py#L156).